README.md

```diff
@@ -120,3 +130,32 @@ Be careful with this method of deployment. `bootRun` is not designed to work wit
 ## External
 
 Deploy resultant `cas/build/libs/cas.war` to a servlet container of choice.
+
+# DEBUG
+
+## Build
+
+```bash
+# shortcut for clean, build and explodeWar (mainly for debug purposes)
+./gradlew[.bat] buildit
```

Comment:
it's probably best if all of this goes into a bash script of some kind like the Maven overlay